engine/processor/src/main/java/io/deephaven/processor/ObjectProcessor.java

```diff
@@ -141,6 +141,15 @@ static ChunkType chunkType(Type<?> type) {
         return ObjectProcessorTypes.of(type);
     }
 
+    /**
+     * The number of outputs. Equivalent to {@code outputTypes().size()}.
+     *
+     * @return the number of outputs
+     */
+    default int size() {
```

Comment:
`numOutputs`? `size()` feels like a name that chooses not to be self-documenting.